src/share/poudriere/common.sh

```diff
@@ -5211,12 +5211,19 @@ ensure_pkg_installed() {
 	[ -z "${force}" ] && [ -x "${mnt}${PKG_BIN}" ] && return 0
 	# Hack, speed up QEMU usage on pkg-repo.
 	if [ ${QEMU_EMULATING} -eq 1 ] && \
-	    [ -f /usr/local/sbin/pkg-static ] && \
-	    for pkg_ext in ${PKG_EXT} txz; do [ -r "${MASTERMNT}/packages/Latest/pkg.${pkg_ext}" ] || continue && injail_ver=$( basename `realpath "${MASTERMNT}/packages/Latest/pkg.${pkg_ext}"` | sed -e 's/pkg-//' -e "s/\.${pkg_ext}//" -e 's/\.pkg//' ); done && \
-	    host_ver=$( /usr/local/sbin/pkg-static -v ) && \
-	    [ ${host_ver} = ${injail_ver} ]; then
-		cp -f /usr/local/sbin/pkg-static "${mnt}/.p/pkg-static"
-		return 0
+	    [ -f /usr/local/sbin/pkg-static ]; then
+		for pkg_ext in ${PKG_EXT} txz; do
+			[ -r "${MASTERMNT}/packages/Latest/pkg.${pkg_ext}" ] || continue
+			injail_ver=$( realpath "${MASTERMNT}/packages/Latest/pkg.${pkg_ext}" )
+			injail_ver=${injail_ver##*/}
+			injail_ver=${injail_ver##*-}
+			injail_ver=${injail_ver%.*}
+			host_ver=$( /usr/local/sbin/pkg-static -v )
+			if [ "${host_ver}" = "${injail_ver}" ]; then
+				cp -f /usr/local/sbin/pkg-static "${mnt}/.p/pkg-static"
+			return 0
```

Comment:
tab

Reply:
> tab

?

Reply:
`return 0` is indented wrong. Needs a tab in front.

Reply:
oh right, fixed.